src/app/components/LogInButton/LogInButton.module.css

```diff
@@ -0,0 +1,10 @@
+.logInButton {
```

Comment:
This is very similar to the `registerButton`. Try to find more generic names for the components like `PrimaryButton` and `SecondaryButton` to allow usage in different scenarios. You might want to reuse this component somewhere else.

I would merge all buttons to a single component and change the CSS BEM like:

```css
.button {
  // Base CSS
}

.button--primary {
  background-color: var(--primary-color);
}

.button--secondary {
  background-color: var(--secondary-color);
}

.button--round {
  border-radius: 50%; 
  // more styles for the round button you like to use as back button
}
```